src/Toolkit/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: build
+build:
+	./bin/build-registry.php \
+		--destination=registry/default \
+		--licenses="MIT" \
+		Symfony "https://www.symfony.com"
```

Comment:
For a 2nd iteration, we must add a CI job that will run this command and check if `git status` is clean, like we did for JS Dist files.